client/me/concierge/calendar-card.js

```diff
@@ -82,7 +83,10 @@ class CalendarCard extends Component {
 	};
 
 	render() {
-		const { times, translate } = this.props;
+		const { isDefaultLocale, times, translate } = this.props;
+		const description = isDefaultLocale
+			? translate( 'Sessions are 30 minutes long.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Sessions are 30 minutes long.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).